code/cfile/cfile.cpp

```diff
@@ -634,13 +635,14 @@ void cf_create_directory( int dir_type )
 
 	} while( current_dir != CF_TYPE_ROOT );
 
-	
 	int i;
 
 	for (i=num_dirs-1; i>=0; i-- )	{
 		cf_create_default_path_string( longname, sizeof(longname)-1, dir_tree[i], NULL );
-		mprintf(( "CFILE: Creating new directory '%s'\n", longname ));
-        mkdir_recursive(longname);
+		if( stat( longname, &statbuf ) != 0) {
```

Comment:
According to Appveyor, `stat` is not valid on Windows so we need a different solution here. Since you already determine the type of the path you could also check if it's a directory and if it isn't then we could issue a log message or a Warning to notify the user that something is wrong.